dotCMS/src/main/java/com/dotmarketing/quartz/job/FreeServerFromClusterJob.java

```diff
@@ -1,24 +1,29 @@
 package com.dotmarketing.quartz.job;
 
+import com.dotcms.cluster.ClusterUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.cluster.ClusterUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)